acbuild/acbuild.go

```diff
@@ -178,21 +188,36 @@ func runWrapper(cf func(cmd *cobra.Command, args []string) (exit int)) func(cmd
 			return
 		}
 
+		defer func() {
+			err = a.End()
+			if err != nil {
+				stderr("%v", err)
+				if cmdExitCode == 0 {
+					cmdExitCode = getErrorCode(err)
+				}
+			}
+		}()
+
 		cmdExitCode = cf(cmd, args)
 
-		err = a.Write(aciToModify, true, false, nil)
+		dir, file := path.Split(aciToModify)
+		tmpACIFile := path.Join(dir, "."+file+".tmp")
```

Comment:
any reason to roll your own tempfile here rather than using ioutil.Tempfile() or similar?